.github/workflows/run_test_suite.yaml

```diff
@@ -78,15 +56,31 @@ jobs:
       - name: 'Run Linter'
         run: cargo clippy --all -- -D warnings
 
-  run-test-suite-linux:
-    runs-on: ubuntu-latest
+  run-rust-test-suite:
+    name: 'Run Rust test suite'
+    strategy:
+      matrix:
+        features: ['test-kubo,headers', 'test-kubo,headers,rocksdb']
+        platform: ['ubuntu-latest', 'windows-latest', 'macos-13']
```

Comment:
We use `macos-12` for the swift tests, should these be synced?

Reply:
Also surprised macos doesn't need any dependencies installed

Reply:
We should update to MacOS 13 if it works.

It's possible that we don't need these dependencies to run the core Rust tests anymore. Or else, that the associated packages already have suitable versions installed on MacOS.